### ISSUE TITLE
Checkout página personalizada

### ISSUE THREAD

**Issue by gotzsys** [deivisonarthur/OSC-Magento-Brasil#147]
Olá, sempre que vou configurar um tema diferente preciso realizar ajustes na parte do checkout devido a incompatibilidades com os temas, versões diferentes da bootstrap... enfim... Até eu ter tempo de desenvolver um tema próprio e mais organizado vou me virando! Pensei então que para resolver isto de uma vez por todas, poderia fazer esta tela isolada de chekout, sem herdar os estilos e configurações do template ativo. O que eu fiz foi editar o arquivo onestepcheckout.xml e alterar o template 1column.phtml para outro arquivo onde configuro o estilo default do tema rwd, limpo e sem conflitos... Minha dúvida é, existe alguma forma mais organizada de fazer isto?

**Comment by rafaelstz** [comment]
Sim, @diogenesnh seguindo as boas práticas no desenvolvimento **Magento** o ideal seria:
- Fazer as alterações relativas a xml no local.xml no template.
- Ao invés de usar outro 1column.phtml editar apenas o bloco do módulo.

Você poderá encontrar as boas prática nesse site [Magento Boas Práticas](http://rafaelstz.github.io/magento-boaspraticas/).

**Comment by antoniokarlos** [comment]
@gotzsys  e @rafaelstz  Será que vocês poderiam me dar uma ajudinha??

Tenho um magento com a versão do OSC 4.0.2 eu coloqueio os arquivos para atualizar para a versão 6.0, no fim. não funcinou a versão 6.0.

Será que vocês poderiam me dar uma dica de como eu configuro o xml para reconhecer a versão 6.0?

Estou utilizando um tema PORTO no magento, não tem esse tema RWD, estou meio que perdido, poderiam me dar uma ajudinha?

Agradeço desde já!
